src/movement.h

```diff
@@ -119,14 +119,18 @@ class MoveEvent final : public Event
 	const std::string& getVocationString() const { return vocationString; }
 	void setVocationString(const std::string& str) { vocationString = str; }
 	uint32_t getWieldInfo() const { return wieldInfo; }
-	const VocEquipMap& getVocEquipMap() const { return vocEquipMap; }
-	void addVocEquipMap(std::string vocName)
+	const VocationEquipSet& getVocationEquipSet() const { return vocationEquipSet; }
```

Comment:
What do you think about returning `const auto&` and getting rid of the alias? One less thing polluting the global namespace.